pyproject.toml

```diff
@@ -55,7 +55,7 @@ classifiers = [
 dependencies = [
   "numpy<1.27,>=1.21",  # required for framework layer and base class logic
   "packaging",  # for estimator specific dependency parsing
```

Comment:
(completely out of scope of this PR)

Should we add some bounds for this as well in a different PR? I never used it myself, but I expect this to be quite stable so should be easy.

Reply:
I think `packaging` is extremely stable, since if it would break, all of the python package ecosystem would break. MAJOR version bound?